alpha/lib/model/entry.php

```diff
@@ -4030,9 +4030,9 @@ public function getObjectParams($params = null)
 			'entitled_kusers_view' => $this->getEntitledKusersViewArray(),
 			'kuser_id' => $this->getKuserId(),
 			'creator_kuser_id' => $this->getCreatorKuserId(),
-			'name' => $this->getName(),
-			'description' => $this->getDescription(),
-			'tags' => explode(',', $this->getTags()),
+			'name' => multiLingualUtils::getElasticFieldValue($this, self::NAME),
+			'description' =>  multiLingualUtils::getElasticFieldValue($this, self::DESCRIPTION),
+			'tags' =>  multiLingualUtils::getElasticFieldValue($this, self::TAGS),
```

Comment:
tags in each language can contain multiple values separated by a comma, how this works in the current implementation?

Reply:
the same way as far as i understand. tags were a string separated with comma, and here exploded into array.
basically now we do the same to all these fields (just pass them as an array instead of exploding).
Is this what you mean?

Reply:
the same way as far as i understand. tags were a string separated with comma, and here exploded into array.
basically now we do the same to all these fields (just pass them as an array instead of exploding).
Is this what you mean?